### PR TITLE
Add comment about `diff` tests

### PR DESCRIPTION
**Pull request by AlekSi**
## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by chilagrow — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM

**Comment by AlekSi** [comment]
@Fashander PTAL

File: website/docs/diff.md [hunk 1]
```diff
@@ -4,15 +4,18 @@ sidebar_position: 6
...
 2. FerretDB does not support NUL (`\0`) characters in strings.
 3. Document keys must not contain `$` sign.
 4. Database and collection names restrictions:
    * name cannot start with the reserved prefix `_ferretdb_`;
    * name must not include non-latin letters, spaces, dots, dollars or dashes;
-   * collection name length must be less or equal than 120 symbols, database name length limit is 63 symbols;
    * name must not start with a number;
-   * database name cannot contain capital letters.
+   * database name cannot contain capital letters;
+   * database name length limit is 63 symbols;
```
Comment (Fashander) [inline website/docs/diff.md:17]:
```suggestion
 * database name length cannot be more than 63 characters.
 * collection name length cannot be more than 120 characters.

```

Comment (Fashander) [reply website/docs/diff.md:17]: The use of characters is preferable to symbols. Symbols can sometimes mean "@#$%^" and so on, which won't be right.

**Review by Fashander — CHANGES_REQUESTED** [review]

**Comment by AlekSi** [comment]
@Fashander PTAL

**Review by Fashander — APPROVED** [review]
LGTM

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by noisersup — APPROVED** [review]

**Pull request merged**